### PR TITLE
Plugin updates

### PR DESCRIPTION
**Pull request by flexiondotorg**
This pull requests updates a few plugins to current releases.

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -983,11 +983,15 @@ parts:
...
     source: https://github.com/nowrep/obs-vkcapture.git
+    source-tag: 'v0.8.1'
     cmake-parameters:
       - -DCMAKE_INSTALL_PREFIX=/usr
       - -DCMAKE_BUILD_TYPE=Release
       - -DCMAKE_C_FLAGS_RELEASE=-s
       - -DCMAKE_CXX_FLAGS_RELEASE=-s
+    override-pull: |
+      snapcraftctl pull
+      sed -i 's|VERSION 3\.19|VERSION 3\.16|' CMakeLists.txt
```
Comment (diddledani) [inline snap/snapcraft.yaml:994]: You could use `build-snaps: [cmake]` to get a more recent version to obviate the need for this...?

File: snap/snapcraft.yaml [hunk 2]
```diff
@@ -983,11 +983,15 @@ parts:
     plugin: cmake
     after: [ obs ]
     source: https://github.com/nowrep/obs-vkcapture.git
+    source-tag: 'v0.8.1'
```
Comment (diddledani) [inline snap/snapcraft.yaml:986]: Any reason `0.9` or `0.9.1` wasn't used?

File: snap/snapcraft.yaml (hunk 1 above)
Comment (flexiondotorg) [reply snap/snapcraft.yaml:994]: A newer `cmake` is not actually required. This reduces build time by not requiring a download of the cmake snap.

File: snap/snapcraft.yaml (hunk 2 above)
Comment (flexiondotorg) [reply snap/snapcraft.yaml:986]: Only that this was the version of `obs-vkcapture` available when the commit was made and the version that I have tested.

**Review by diddledani — APPROVED** [review]

**Pull request merged**